.circleci/config.yml

```diff
@@ -79,11 +80,11 @@ jobs:
             echo $QUAY_TOKEN | docker login -u=$QUAY_USER --password-stdin quay.io
 
       - run:
-          name: Build 0.5.0-drupal87 image
+          name: Build 0.6.0-drupal89 image
           command: |
-            git checkout 0.5.0-drupal87
-            docker build -t quay.io/deviantintegral/drupal_tests:0.5.0-drupal87 . && \
-            docker push quay.io/deviantintegral/drupal_tests:0.5.0-drupal87
+            git checkout 0.6.0-drupal89
```

Comment:
Will need to remember to tag `drupal_tests` after this PR is merged, and then either manually run `build_and_push` or uncomment the cron schedule (eg. https://github.com/deviantintegral/drupal_tests/blob/master/.circleci/config.yml#L95-L104)

Reply:
This is done and the job succedded: 

https://app.circleci.com/pipelines/github/deviantintegral/drupal_tests/23/workflows/52200dba-6db9-4324-bd93-8a9b69688ca7/jobs/526
https://quay.io/repository/deviantintegral/drupal_tests?tab=tags